rpa-home/modules/ROOT/pages/automation-usecase-legacysystemdataextraction-extractfrompdf.adoc

```diff
@@ -0,0 +1,64 @@
+# Finding Information in a PDF
+// number the headings to correspond to the numbered list of steps
+:sectnums:
+
+First, you need to find the order ID in the PDF.
```

Comment:
The short description should summarize the entire topic and give some user value.

The first step in extracting data from a legacy system is to find the order ID in the PDF.